ppu/display.go

```diff
@@ -0,0 +1,55 @@
+package ppu
+
+type display struct {
+	brightness  uint8 // Display brightness
+	forceBlank  bool  // If true, force screen to blank
+	vScanning   bool  // If true, interlace mode
+	objVDisplay bool  // \
+	bgVDisplay  bool  //  \
+	hPseudoMode bool  //    Some dank parameters
+	ExtBgMode   bool  //  /
+	ExtSynchro  bool  // /
+}
+
+// 2100h - INIDISP - Display Control 1 (W)
+func (ppu *PPU) inidisp(data uint8) uint8 {
+	ppu.display.brightness = data & 0x0F
+	ppu.display.forceBlank = data&0x80 != 0
+	return 0
+}
+
+// 212Ch - TM - Main Screen Designation (W)
+func (ppu *PPU) tm(data uint8) uint8 {
+	for i := 0; i < 4; i++ {
+		ppu.backgroundData.bg[i].mainScreen = data&0x01 != 0
+		data = data >> 1
+	}
+	ppu.oam.mainScreen = data&0x01 != 0
+	return 0
+}
+
+// 212Dh - TS - Sub Screen Designation (W)
+func (ppu *PPU) ts(data uint8) uint8 {
+	for i := 0; i < 4; i++ {
+		ppu.backgroundData.bg[i].subScreen = data&0x01 != 0
+		data = data >> 1
+	}
+	ppu.oam.subScreen = data&0x01 != 0
+	return 0
+}
+
+// 2133h - SETINI - Display Control 2 (W)
+func (ppu *PPU) setini(data uint8) uint8 {
```

Comment:
I think it would be better to directly use 0x02, 0x04, 0x08, etc. Instead of shifting data

Reply:
Indeed but I think we should keep the loop

Reply:
which loop ?

Reply:
the for loop

Reply:
Ah ! yes i was only speaking of setini register

Reply:
done